bot.js

```diff
@@ -16,6 +16,7 @@ const help = require('./help/index');
 
 client.on("ready", _ => {
     console.log(`Logged in as ${client.user.tag}!`);
+    client.channels.get('499131225835962382').send('Did you miss me?');
```

Comment:
What's the number in get()?

Reply:
It's the channel ID hard coded. Will improve before merge.